teams/team-geordie/dev/ho11y/templates/ho11y.yaml

```diff
@@ -31,7 +31,7 @@ spec:
         - name: OTEL_RESOURCE_ATTRIB
           value: "frontend"
         - name: OTEL_EXPORTER_OTLP_ENDPOINT
-          value: "adot:4317"
```

Comment:
So during the demo (to the SI) I was presenting the teams concept and discovered that ho11y app was not there. I suppose that was potentially the reason. Let's think about a test (e.g. our conformance test) that we can run along with the blueprint to make sure no pods/deployments/daemonsets are in the failed state. 
I wonder if it will be too much effort to shape that test as a container as well and run it as a k8s job. Then we can just test out that job and rerun at any point.